frame/lending/src/lib.rs

```diff
@@ -776,8 +777,18 @@ pub mod pallet {
 				borrowers.len() <= T::MaxLiquidationBatchSize::get() as usize,
 				Error::<T>::MaxLiquidationBatchSizeExceeded
 			);
-			Self::liquidate_internal(&sender, &market_id, borrowers.clone())?;
-			Self::deposit_event(Event::LiquidationInitiated { market_id, borrowers });
+			// Unwrapped because of upper check.
+			// Perhaps borrowers ids should be in the bounded vector.
+			let borrowers = BoundedVec::<_, T::MaxLiquidationBatchSize>::try_from(borrowers)
```

Comment:
ok_or_err ?

Reply:
in theory you could use oneline with BiBoundedVec instead of 3 line checks.

also you do not check lower len > 0 (so at least one elemement in vector).

Reply:
BoundedVec used here because of liquidate_internal() function. It uses BoundedVec to avoid explicit MaxLiquidationBatchSize checking. Using BiBoundedVec in the liquidate_interanl() looks restrictive since potential user of this function may want to pass empty vector of borrowers there (for some reason). Perhaps we can use BoundedVec in dispatchable liquidate() function too.

Reply:
Regarding unwrap_or_default(), the same situation. Would not it be too restrictive emit en error because of empty borrowers vector?